.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,5 @@
-Make sure you have checked _all_ steps below.
-
-### Jira
-
-- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
-  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
-  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
-  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
-  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
```

Comment:
Same here. I think this is something useful that we should keep it here to inform users.

Reply:
Agree!. Added licence information (also as a note at the bottom without checkmark).